spec/features/admin_can_create_products_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+feature "an admin can create products" do
+  before do
+    User.create(first_name: "Mike",
+                last_name: "Dorrance",
+                email: "mike@mike.com",
+                password: "12345678",
+                role: 1)
+
+    visit "/"
+    click_link "Login"
+    fill_in "Email", with: "mike@mike.com"
+    fill_in "Password", with: "12345678"
+    click_button "Login"
```

Comment:
We could stub #current_user out instead of using capybara to log the user in. It's faster! What do you all think?